viewer/src/main/webapp/viewer-html/components/AttributeList.js

```diff
@@ -397,8 +409,13 @@ Ext.define ("viewer.components.AttributeList",{
         var attributeList = new Array();
         var columns = new Array();
         var index = 0;
+        var hasGeometry = false;
+        var geometryTypes = ["geometry", "point", "multipoint", "linestring", "multilinestring", "polygon", "multipolygon"];
         for(var i= 0 ; i < attributes.length ;i++){
             var attribute = attributes[i];
+            if(geometryTypes.indexOf(attribute.type) !== -1) {
+                hasGeometry = true;
+            }
```

Comment:
probably better to check if the feature type/layer has a `geometryAttribute` or `geometryAttributeIndex` the type is not that interesting